### PR TITLE
fix: support disabling options in multi-select

### PR DESCRIPTION
**Pull request by arjunlalb**
## Description
support disabling options in multi-select

### Testing
Manually verified. UTs added.

### Checklist:
- [x] My changes generate no new warnings
- [x I have added tests that prove my fix is effective or that my feature works
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: projects/components/src/multi-select/multi-select.component.scss [hunk 1]
```diff
@@ -92,7 +92,7 @@
   flex-direction: column;
 
   .divider {
-    padding: 0px 16px 4px 16px;
+    padding: 0 16px 4px 16px;
```
Comment (anandtiwary) [inline projects/components/src/multi-select/multi-select.component.scss:95]: why remove px ?

File: projects/components/src/multi-select/multi-select.component.scss [hunk 2]
```diff
@@ -107,8 +107,13 @@
...
     align-items: center;
 
+    &.disabled {
+      color: $gray-5;
+      cursor: not-allowed;
+    }
+
     .checkbox {
-      margin: 0px;
+      margin: 0;
```
Comment (anandtiwary) [inline projects/components/src/multi-select/multi-select.component.scss:116]: same question here

**Review by anandtiwary — COMMENTED** [review]
Looks good apart from the one question

File: projects/components/src/multi-select/multi-select.component.scss (hunk 1 above)
Comment (arjunlalb) [reply projects/components/src/multi-select/multi-select.component.scss:95]: zeroes don't need px :)

Comment (anandtiwary) [reply projects/components/src/multi-select/multi-select.component.scss:95]: so this is just a nit change?

**Review by anandtiwary — APPROVED** [review]

File: projects/components/src/multi-select/multi-select.component.scss (hunk 1 above)
Comment (arjunlalb) [reply projects/components/src/multi-select/multi-select.component.scss:95]: yea. this is just a minor cleanup since I was in the area :)

**Pull request merged**